com.examples.exercise/src/main/java/com/examples/exercise/SchoolController.java

```diff
@@ -0,0 +1,17 @@
+package com.examples.exercise;
+
+import java.util.List;
+
+public class SchoolController {
+	private Database database;
+
+	public SchoolController(Database database) {
+		super();
+		this.database = database;
+	}
+
+	public List<Student> getAllStudents() {
+		return database.getAllStudentsList();
+	}
+
```

Comment:
During merging, you removed the implementation of update of https://github.com/LorenzoBettini/example.exercise/pull/19/files